platform-modules/batch-models/src/main/scala/org/ekstep/analytics/job/TelemetryReplayJob.scala

```diff
@@ -0,0 +1,55 @@
+package org.ekstep.analytics.job
+
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.ekstep.analytics.framework._
+import org.ekstep.analytics.framework.util.{CommonUtil, JSONUtils, JobLogger}
+
+import scala.collection.Map
+
+object TelemetryReplayJob extends optional.Application with IJob {
```

Comment:
The name of the class needs to be generic as it will be used for both Telemetry events and Summary events.